docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -42,6 +42,35 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+* *Stereotyping harms* can occur when a system suggests completions which
+  perpetuate stereotypes.
+  These are often seen when search engines propose completions to partially
+  typed queries.
+  See :footcite:cts:`umojanoble2018algorithmsoppression` for an in-depth
+  look at this issue.
+  Note that even stereotypes which are nominally positive are also
+  problematic, since they still create expectations based on outward
+  characteristics, rather than treating people as individuals.
+
+* *Erasure harms* can occur when a system behaves as if groups (or their
```

Comment:
Please remind me of others I've missed.... and I'm pretty sure there's a paper we should be referencing here, in addition to the YouTube video.

Reply:
The paper is "Solon Barocas, Kate Crawford, Aaron Shapiro, Hanna Wallach, 2017 'The Problem With Bias: Allocative Versus Representational Harms in Machine Learning', SIGCIS Conference", but strangely the paper is not available anywhere online...

Reply:
Thanks. I've managed to track down a BibTeX entry for it at least, and have emailed Solon to ask if there is a version online somewhere. Or something else he'd suggest.

Reply:
If I recall correctly this is not a paper but a keynote, which is probably why its hard to locate. Ive asked @hannawallach  about this before, she noted I could cite the keynote by Kate and possibly also "Language (Technology) is Power" paper by Su Lin Blodgett et al.  (which was seconded by Solon).

Reply:
I agree that that paper (https://arxiv.org/abs/2005.14050 for reference) is definitely one I should work into this section. However, it doesn't seem to have a listing of harms like I'm building here. I'd like it if we could cite something which can be rendered in dead tree format as well; WiFi doesn't tend to work on the train home.

Reply:
That certainly lists some (and is another paper we should be figuring out how to include), but it's still not quite what I meant. The list of harms is also in the appendix, not the main body of the paper.